cmd/capkctl/main.go

```diff
@@ -0,0 +1,1257 @@
+// Copyright 2019 VMware, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+package main
```

Comment:
This is testing and can be skipped. It's a helper to set up capk. Right now it's CRDs generated from kustomzie and copy-pasted into strings.

The one trick is to look at the container mounts.